content/en/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Check whether Dockershim deprecation affects you
+content_type: task 
+reviewers:
+- SergeyKanzhelev
+weight: 20
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.20" state="deprecated" >}}
```

Comment:
Does the feature state shortcode apply in this case?

Reply:
@sftim can you suggest?

Reply:
I wouldn't use the shortcode here.